stackit/internal/services/rabbitmq/instance/resource_test.go

```diff
@@ -62,7 +62,87 @@ func TestMapFields(t *testing.T) {
 				ImageUrl:           types.StringValue("image"),
 				CfOrganizationGuid: types.StringValue("org"),
 				Parameters: types.ObjectValueMust(parametersTypes, map[string]attr.Value{
-					"sgw_acl": types.StringValue("acl"),
+					"sgw_acl":                types.StringValue("acl"),
+					"consumer_timeout":       types.Int64Null(),
+					"enable_monitoring":      types.BoolNull(),
+					"graphite":               types.StringNull(),
+					"max_disk_threshold":     types.Int64Null(),
+					"metrics_frequency":      types.Int64Null(),
+					"metrics_prefix":         types.StringNull(),
+					"monitoring_instance_id": types.StringNull(),
+					"plugins":                types.ListNull(types.StringType),
+					"roles":                  types.ListNull(types.StringType),
+					"syslog":                 types.ListNull(types.StringType),
+					"tls_ciphers":            types.ListNull(types.StringType),
+					"tls_protocols":          types.StringNull(),
+				}),
```

Comment:
I would write these in a `default_params_values` test case, and then the params with the values from the test below could be moved here (and `simple_values_params` could be deleted)

Reply:
Same for tests below

Reply:
Good point, I agree